frontend/app/dashboard/src/views/dashboard/webshop/edit/products/EditProductPriceView.vue

```diff
@@ -59,11 +60,10 @@ function save() {
 }
 
 async function deleteMe() {
-    if (!await CenteredMessage.confirm('Ben je zeker dat je deze prijskeuze wilt verwijderen?', 'Verwijderen')) {
+    const p = await useDeleteProductPrice(props.product, props.productPrice);
```

Comment:
```
const deleteProductPrice = useDeleteProductPrice();
async function deleteMe() {
    const p = await deleteProductPrice(props.product, props.productPrice);
```

Een hook moet je eigenlijk altijd rechtstreeks aanroepen binnen de 'setup' van je Vue component, anders kan het geen andere hooks aanspreken. Een alternatief zijn om zoals je nu doet een losse functie te schrijven, maar dan gebruik je best niet de prefix 'use' om geen verwarring te creeren.

Ookal heb je in die hook nu geen andere hooks nodig, het is sowieso op langere termijn flexibeler om met hooks te werken. Dus de extra boilerplate code is het meestal nu wel waard, en schept ook een patroon dat herkenbaar is

Reply:
In dit geval zou `useDeleteProductPrice` dus een functie moeten returnen :)